sourcemash/api/feeds.py

```diff
@@ -109,7 +123,7 @@ def post(self):
 class FeedListAllAPI(Resource):
 
     def get(self):
-        return {'feeds': [marshal(feed, feed_status_fields) for feed in Feed.query.all()]}
+        return {'feeds': [marshal(feed, feed_status_fields) for feed in Feed.query.filter(Feed.public==True).all()]}
```

Comment:
This means that if newly added feeds default to public=False, then they'll never show up on the Browse page for that user. This is a big issue.